test/tests/outputs/debug/tests

```diff
@@ -35,14 +35,23 @@
      max_parallel = 1
    [../]
 
-   [./show_top_residuals_debug]
+   [./show_top_residuals_debug_nodal]
      # Test that top residuals are displayed using DebugOutput object via Debug block
      type = RunApp
      input = show_top_residuals_debug.i
      expect_out = "\[DBG\]\[\d+\].*'u' in subdomain\(s\) {'block_zero', 'block_one'}.*at node"
      max_parallel = 1
    [../]
 
+   [./show_top_residuals_debug_elemental]
+     # Test that top residuals are displayed using DebugOutput object via Debug block
+     type = RunApp
+     input = show_top_residuals_debug.i
+     cli_args = 'Variables/u/family=MONOMIAL Variables/u/order=CONSTANT'
+     expect_out = "\[DBG\]\[\d+\].*'u' in subdomain\(s\) {'block_one'}.*at element"
+     max_parallel = 1
+   [../]
```

Comment:
Idea: How about adding an elemental variable into the input file, so that we test both types at the same time?